### PR TITLE
Pr displ manager

### PR DESCRIPTION
**Pull request by andr2000**

### CODE REVIEW

**Comment by al1img** [comment]
Reviewed-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>

**Review by lorc — APPROVED** [review]
I would love to see detailed commit messages. But, anyways:

`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by andr2000** [comment]
@iartemenko pls merge

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/display-manager.service [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=EPAM Display Manager
+Requires=dbus.service
+Before=HomeScreen.service
+
+[Service]
+Environment="XDG_RUNTIME_DIR=/run/user/0"
+Type=dbus
+BusName=com.epam.DeviceManager
```
Comment (al1img) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/display-manager.service]: Change to com.epam.DisplayManager

**Pull request merged**